src/gpu/nvidia/cudnn_binary.cpp

```diff
@@ -33,19 +34,21 @@ status_t cudnn_binary_t::execute(const exec_ctx_t &ctx) const {
             = utils::downcast<nvidia::sycl_cuda_stream_t *>(ctx.stream());
 
     return cuda_stream->interop_task([&](::sycl::handler &cgh) {
-        auto src_0_acc = CTX_IN_ACCESSOR(DNNL_ARG_SRC_0);
-        auto src_1_acc = CTX_IN_ACCESSOR(DNNL_ARG_SRC_1);
-        auto dst_acc = CTX_OUT_ACCESSOR(DNNL_ARG_DST);
+        cgh.depends_on(cuda_stream->get_deps());
```

Comment:
It looks like we can take advantage of having [sycl_cuda_stream_t::interop_task](https://github.com/oneapi-src/oneDNN/blob/495220cf9459502bca8c103cada1ed0c430a6d08/src/gpu/nvidia/sycl_cuda_stream.cpp#L101) and put `cgh.depends_on(this->get_deps());` in one place there. What do you think?

Reply:
Makes sense. Done.

Reply:
Thanks!